command/ca/provisionerbeta/list.go

```diff
@@ -0,0 +1,66 @@
+package provisionerbeta
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/pki"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/urfave/cli"
+)
+
+func listCommand() cli.Command {
+	return cli.Command{
+		Name:   "list",
+		Action: cli.ActionFunc(listAction),
+		Usage:  "list provisioners configured in the CA",
+		UsageText: `**step ca provisioner list**
+[**--ca-url**=<uri>] [**--root**=<file>]`,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "ca-url",
+				Usage: "<URI> of the targeted Step Certificate Authority.",
+			},
+			cli.StringFlag{
+				Name:  "root",
+				Usage: "The path to the PEM <file> used as the root certificate authority.",
+			},
+		},
+		Description: `**step ca provisioner list** lists the provisioners configured
+in the CA.
+
+## EXAMPLES
+
+Prints a JSON list with active provisioners:
+'''
+$ step ca provisioner list
+'''`,
+	}
+}
+
+func listAction(ctx *cli.Context) error {
```

Comment:
For a future list action I think it should be good to have the option of `--format human`, I would say `--raw` or `--json` for the old one, but it won't be backward compatible (probably not that important).

The `human` format or default format if we make it backward compatible can be:

```text
TYPE    NAME
JWK     admin
OIDC    Google
```